code/game/machinery/doors/airlock.dm

```diff
@@ -70,6 +71,7 @@
 	var/obj/item/electronics/airlock/electronics
 	COOLDOWN_DECLARE(shockCooldown)
 	var/obj/item/note //Any papers pinned to the airlock
+	var/obj/item/seal //The seal on the airlock
```

Comment:
This needs to be documented as per our [documentation guidelines](https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md)

Reply:
I  think  that's resolved now